### DISCUSSION TITLE
Require `go get github.com/99designs/gqlgen`, after `go run github.com/99designs/gqlgen`

### DISCUSSION THREAD

**Discussion by n9te9** [99designs/gqlgen#2152] [Q&A]
### What happened?
Certainly, I'm required `go get github.com/99designs/gqlgen` after running `go run github.com/99designs/gqlgen`.

1. run `go run github.com/99designs/gqlgen` → ok
2. run `go run github.com/99designs/gqlgen` → require `go get` like below message.
```
missing go.sum entry for module providing package github.com/urfave/cli/v2 (imported by github.com/99designs/gqlgen); to add: go get github.com/99designs/gqlgen@v0.17.5
```
3. run `go get github.com/99designs/gqlgen@v0.17.5` → ok
4. run `go run github.com/99designs/gqlgen` → ok
5. → 2.

### What did you expect?
Once, I run `go get github.com/99designs/gqlgen`, I don't want to be required `go get` other than the time that I want to upgrade modules.

### versions
 - `go run github.com/99designs/gqlgen version`?  v0.17.5
 - `go version`? 1.18.1

**Comment by rlch** [comment]
Any solution to this? I'm experiencing the same issue on:
- `gqlgen v0.17.9`
- `go 1.18.3 darwin/arm64`

**Comment by rlch** [comment]
I've recently been using the `gqlgen` binary directly to generate. Try removing the package from `go.mod` and then run:

`go install github.com/99designs/gqlgen`

**Comment by sukinoverse** [comment]
I've experienced the same issue after upgrade from v0.16.0 to v0.17.12. Any solution yet?
